app/theme/client/imports/components/message-box.css

```diff
@@ -144,6 +144,12 @@
 		& .rc-input__icon-svg--plus {
 			transition: transform 0.1s linear;
 		}
+		&:hover,
+		&:focus,
+		&.active {
+			color: red;
+			transform: scale(1.5);
```

Comment:
I don't think its a good idea to use scaling property, It doesn't go well with UI/UX. This is just my opinion. Other places we are doing linear transform, maybe we can do something like that.

Reply:
We can use linear transform too, but here we would like to keep the time for transition very small, so that's why I used scaling property.

Reply:
Hey @yash-rajpal,
So any other transition except zoom you have in mind? I'll just do it and make the final commit